shared/chat/conversation/header-area/normal/container.tsx

```diff
@@ -61,8 +61,8 @@ const HeaderBranch = (props: Props) => {
 
 export default Container.withSafeNavigation(
   Container.connect(mapStateToProps, mapDispatchToProps, (stateProps, dispatchProps) => ({
-    badgeNumber: stateProps._badgeMap.reduce(
-      (res, currentValue, currentConvID) =>
+    badgeNumber: [...stateProps._badgeMap.entries()].reduce(
```

Comment:
this seems to do twice as much work to make the array and then reduce on it. It doesn't seem like there's a way to do it as cleanly without that, but I figured I should point it out and make sure it's ok since we're working on chat performance lately.

Reply:
This is very fast